app/src/features/charts/MapNavChart.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect } from "react";
+import UsaChloroplethMap from "../charts/UsaChloroplethMap";
+import { USA_FIPS, USA_DISPLAY_NAME, STATE_FIPS_MAP } from "../../utils/Fips";
+import Alert from "@material-ui/lab/Alert";
+import Breadcrumbs from "@material-ui/core/Breadcrumbs";
+import Link from "@material-ui/core/Link";
+import Typography from "@material-ui/core/Typography";
+import { VariableId } from "../../utils/variableProviders";
+
+function GeographyBreadcrumb(props: {
+  text: string;
+  isClickable: boolean;
+  onClick?: () => void;
+}) {
+  return (
+    <>
+      {props.isClickable && (
+        <Link color="inherit" onClick={() => props.onClick!()}>
+          {props.text}
+        </Link>
+      )}
+      {!props.isClickable && (
+        <Typography color="textPrimary">{props.text}</Typography>
+      )}
+    </>
+  );
+}
+
+function MapNavChart(props: {
+  varField: VariableId;
+  varFieldDisplayName: string;
+  fipsGeo: string;
+  countyFips: string | undefined;
+  data: Record<string, any>[];
+  updateGeoCallback: (message: string) => void;
+}) {
+  const [countyName, setCountyName] = useState<string>();
```

Comment:
Maybe I'm missing something, but I'm having a hard time understanding why countyName is local state and also countyFips can be passed in.

What do you think about lifting all this state up to the parent and just using the fipsGeo prop, which could be state or county?